operator/src/test/resources/token-test-realm.yaml

```diff
@@ -0,0 +1,1725 @@
+apiVersion: keycloak.org/v2alpha1
```

Comment:
Can't we re-use the example realm instead?

Reply:
Unfortunately not, we are relying on the configuration of this specific hand crafted realm to run the test.

Also this is increasing the coverage for `RealmImport` CRs 🙂

Reply:
👍